config

```diff
@@ -419,6 +419,7 @@ CONFIG_MULTI_IRQ_HANDLER=y
 CONFIG_ARM_ERRATA_818325_852422=y
 CONFIG_ARM_ERRATA_821420=y
 CONFIG_ARM_ERRATA_825619=y
+CONFIG_ARM_ERRATA_CR711784=y
```

Comment:
`CONFIG_ARM_ERRATA_CR711784` should be below `CONFIG_ARM_ERRATA_FOOBAR` - otherwise, there will always be a difference between .config before and after the build (and we want there to be zero difference, to know that the kernel configuration is fine)